dist/js/autocomplete.js

```diff
@@ -421,6 +421,15 @@
         },
 
 		/*
+			<property:searchkeys>
+				<desc>Array of object fields to be searched.</desc>
+			</property:searchkeys>
+		*/
+		get searchkeys () {
```

Comment:
What we should do here is the splitting logic. Something like:

return this.config.searchkeys.replace(/\s/g, '').split(',');

This will hand you back a clean array to be used in the filterData() method.